src/modules/manage-team/ManageTeamPage.module.css

```diff
@@ -7,70 +7,83 @@
   gap: 2rem;
 }
 
+.container > div:nth-child(2) {
```

Comment:
This can't be resolved with a style prop or classname on the specified cards? if a card now changes its main tag this will break silently